outlook/calendar/credential/tool.gpt

```diff
@@ -9,5 +9,6 @@ Share Credential: ../../../oauth2 as outlook.calendar.write
             Group.Read.All
             Group.ReadWrite.All
             GroupMember.Read.All
-            User.Read offline_access" as scope
+            User.Read
+            MailboxSettings.Read offline_access" as scope
```

Comment:
small nit, but why do we have `offline_access` on the same line as another permission here when all the others have newlines?

Reply:
Good question. I must have missed that when I first added this credential tool.

will fix

Reply:
fixed